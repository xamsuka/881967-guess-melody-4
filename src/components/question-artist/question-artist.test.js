import React from 'react';
import renderer from 'react-test-renderer';
import QuestionArtistScreen from './question-artist';

const question = {
  type: `artist`,
  song: {
    audioSrc: `https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg`,
    artist: ``
  },
  artists: [
    {
      author: `Пелагея`,
      imageSrc: `https://api.adorable.io/avatars/128`,
    },
    {
      author: `Моргенштерн`,
      imageSrc: `https://api.adorable.io/avatars/128`,
    },
    {
      author: `Элджей`,
      imageSrc: `https://api.adorable.io/avatars/128`,
    },
  ]
};

test(`<QuestionArtist /> render`, () => {
  const three = renderer
    .create(<QuestionArtistScreen question = {question} onAnswerButtonClick = {() => {}}/>).toJSON();

  expect(three).toMatchSnapshot();
});
