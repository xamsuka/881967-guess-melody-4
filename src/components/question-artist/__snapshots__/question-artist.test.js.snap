// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuestionArtist /> render 1`] = `
<section
  className="game__screen"
>
  <h2
    className="game__title"
  >
    Кто исполняет эту песню?
  </h2>
  <div
    className="game__track"
  >
    <div
      className="track"
    >
      <button
        className="track__button track__button--play"
        onClick={[Function]}
        type="button"
      />
      <div
        className="track__status"
      >
        <audio
          src="https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg"
        />
      </div>
      
    </div>
  </div>
  <form
    className="game__artist"
    onClick={[Function]}
  >
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        defaultValue="answer-0"
        id="answer-0"
        name="answer"
        type="radio"
      />
      <label
        className="artist__name"
        htmlFor="answer-0"
      >
        <img
          alt="Пелагея"
          className="artist__picture"
          src="https://api.adorable.io/avatars/128"
        />
        Пелагея
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        defaultValue="answer-1"
        id="answer-1"
        name="answer"
        type="radio"
      />
      <label
        className="artist__name"
        htmlFor="answer-1"
      >
        <img
          alt="Моргенштерн"
          className="artist__picture"
          src="https://api.adorable.io/avatars/128"
        />
        Моргенштерн
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        defaultValue="answer-2"
        id="answer-2"
        name="answer"
        type="radio"
      />
      <label
        className="artist__name"
        htmlFor="answer-2"
      >
        <img
          alt="Элджей"
          className="artist__picture"
          src="https://api.adorable.io/avatars/128"
        />
        Элджей
      </label>
    </div>
  </form>
</section>
`;
